code/game/objects/items/weapons/surgery_tools.dm

```diff
@@ -74,6 +75,28 @@
 /obj/item/weapon/cautery/is_hot()
 	return 1
 
+/obj/item/weapon/cautery/laser
```

Comment:
I still do not see the point of having this type if you never intend to spawn it.
Why not simply make this type the tier 1 model, and have `/obj/item/weapon/cautery/laser/tier2` inherit from it and override the few vars that are different?

Reply:
Fair enough, I just left it that way since it was like that to start.

Reply:
I searched the laser scalpel and it seems that the laser scalpel is in a dmm. Id rather not touch it if it's in mapping.

Reply:
Conveniently for you, the laser scalpel which is in the Research facility vault is a tier 2 scalpel, which you don't need to change the path of. No editing of the map would be necessary.